presto-product-tests/conf/presto/etc/multinode-kerberized-master.properties

```diff
@@ -0,0 +1,31 @@
+#
+# WARNING
+# ^^^^^^^
+# This configuration file is for development only and should NOT be used be
+# used in production. For example configuration, see the Presto documentation.
+#
+
+node.id=will-be-overwritten
+node.environment=test
+node.internal-address-source=FQDN
+
+coordinator=true
+node-scheduler.include-coordinator=true
+query.max-memory=1GB
+query.max-memory-per-node=512MB
+discovery-server.enabled=true
+discovery.uri=https://presto-master.docker.cluster:7778
+
+http.authentication.krb5.config=/etc/krb5.conf
+http-server.authentication.type=KERBEROS,CERTIFICATE
```

Comment:
so here `certificate` mode will be used, correct? Can we try with just `KERBEROS` here since the previous PR https://github.com/prestodb/presto/pull/9611 already handled/tested the mixed multiple auth mode?

Reply:
then internal communication is unsecured. This way makes multionde-kerberos to be coherent with singlenode-kerberos